scripts/install_docker.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ex
+
+curl -sSL https://get.docker.com/ubuntu/ | sh
```

Comment:
It worked for me, when doing `vagrant up` from a blank VM.

Are you sure your vagrant arch is `amd64`?